core/frontend/render-mode-manager.php

```diff
@@ -58,14 +59,14 @@ private function set_current_render_mode() {
 
 		$key = null;
 		$nonce = null;
-		$nonce_action = str_replace( '{post_id}', $post_id, Render_Mode_Interface::NONCE_ACTION_PATTERN );
+		$nonce_action = str_replace( '{post_id}', $post_id, Render_Mode_Base::NONCE_ACTION_PATTERN );
```

Comment:
Use a method to replace here and on `core/frontend/render-modes/render-mode-base.php:24`